src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -137,7 +141,6 @@ export default defineComponent({
       if (this.selectedCourse == null) return;
       this.$emit('add-course', this.selectedCourse, this.selectedRequirementID);
       this.reset();
-      this.closeCurrentModal();
```

Comment:
After we add course, the add modal is still open but it should actually close once the course is added. Can we call `this.escape()`?

https://user-images.githubusercontent.com/54298311/117295146-7ab93f00-ae41-11eb-8bf4-25fda0de3d5b.mov